IPython/html/static/base/less/page.less

```diff
@@ -5,16 +5,23 @@
  */
 
 
+html, body {
+    min-height: 100%;
+    height: 100%;
+    overflow: hidden; // Prevents scrollbar in html element
+}
+
 body {
     background-color: @body-bg;
-    /* This makes sure that the body covers the entire window and needs to
-       be in a different element than the display: box in wrapper below */
-    position: absolute;
-    left: 0px;
-    right: 0px;
-    top: 0px;
-    bottom: 0px;
```

Comment:
Yup, so apparently it's not equivalent. Let's restore it, and remove the `container_` bit.